src/noframe.js

```diff
@@ -0,0 +1,40 @@
+export default function noframe(target, container) {
+  let els = typeof target === 'string' ? document.querySelectorAll(target) : target;
+  if (!('length' in els)) {
+    els = [els];
+  }
+  for (let i = 0; i < els.length; i++) {
+    const el = els[i];
+    const height = el.offsetHeight;
+    const width = el.offsetWidth;
+    let maxwidth = `${width}px`;
```

Comment:
`let` and `const` can be defined within logic blocks: http://wesbos.com/let-vs-const/

I think it's better form to define ` const maxwidth` in your `if` and another one in your `else`  since they are different in each block.